content/doc/book/security/access-control/permissions.adoc

```diff
@@ -141,6 +234,33 @@ Learn more in jep:223[].
 NOTE: This permission was added in Jenkins 2.222.
 Some features, especially those provided by plugins, may not yet support this permission.
 
+=== Credentials permissons
```

Comment:
Should not be placed here as it's not an optional permission as explained in the introductory paragraph.

Also, while Credentials Plugin is as much of a "core" plugin as is possible, what's the threshold for putting plugin docs here?